profiles/tests/test_saved_company.py

```diff
@@ -29,20 +27,22 @@ def test_add_company_to_saved_authenticated(self):
         response = self.client.post(
             path="/api/saved-list/",
             data={
-                "company_pk": "{profile_id}".format(profile_id=self.profile.id)
+                "user": self.user.id,
+                "company": "{profile_id}".format(profile_id=self.profile.id),
             },
         )
-        self.assertEqual(status.HTTP_200_OK, response.status_code)
-        company_added_info = response.data["Company added"]
-        self.assertEqual(company_added_info["company"], self.profile.id)
+        self.assertEqual(201, response.status_code)
```

Comment:
`self.assertEqual(status.HTTP_201_CREATED, response.status_code)`

(`status` should be imported somewhere from `rest_framework`).

Reply:
You mean to return 'self.assertEqual(status.HTTP_200_OK, response.status_code)'?

Reply:
@BelousSofiya in this case you have status `201`, not `200`. So I guess my suggestion above was correct.

The main point is to use `status` constants instead of just 'magic numbers' with response codes.

Reply:
Thank you for feedback. Done